src/python/pants/core/goals/package.py

```diff
@@ -72,11 +72,14 @@ class OutputPathField(StringField, AsyncFieldMixin):
     def value_or_default(self, *, file_ending: str | None) -> str:
```

Comment:
Re testing: this would be good to unit test in a new file `package_test.py`. We shouldn't have done that for a long time, bad on us